src/record-service.js

```diff
@@ -201,7 +201,7 @@ var RecordService = function(provider, type, id) {
     _data.lastUpdatedAt = new Date();
     var dataKey;
     for (dataKey in _data) {
-      if (!_data[dataKey]) {
+      if (typeof _data[dataKey] == 'undefined') {
```

Comment:
@Acefox17 not a huge deal, but I prefer `===` to `==` because [it's more strict](http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons).